sdk/textanalytics/Azure.AI.TextAnalytics/README.md

```diff
@@ -0,0 +1,119 @@
+# Azure Cognitive Services Text Analytics client library for .NET
+Azure Cognitive Services Text Analytics is a cloud service ithat provides advanced natural language processing over raw text. and includes six main functions: sentiment analysis, key phrase extraction, named entity recognition, linked entity recognition, recognition of personally identifiable information, and language detection.
+
+[Source code][textanalytics_client_src] | [Package (NuGet)][textanalytics_nuget_package] | [API reference documentation]() | [Product documentation][textanalytics_docs] | [Samples][textanalytics_samples]
+
+## Getting started
+
+### Install the package
+Install the Azure Text Analytics client library for .NET with [NuGet][nuget]:
+
+```PowerShell
+Install-Package Azure.AI.TextAnalytics
```

Comment:
Do you want to have a specific version for the package?

Reply:
I followed KeyVault's lead here, and they don't have it.

Reply:
Ok, I'll update AzConfig as well.

Reply:
I'd imagine it's so we don't have to reversion the README every time.